mods/Fifty.ServerTeamBalance/mod/scripts/vscripts/fstb_command_callbacks.nut

```diff
@@ -12,33 +12,35 @@ void function FSTM_CommandCallback_Switch( entity player, array< string > args )
 		return
 	}
 
-	entity target = player
-
+	entity target
 	if( FSA_IsAdmin( player ) && args.len() != 0 ) {
-		bool foundTarget = false
 		foreach( entity p in GetPlayerArray() ) {
 			if( p.GetPlayerName().tolower().find( args[0].tolower() ) != -1 ) {
+				if( target != null ){
```

Comment:
What if the full name was provided, but we still reach this?
Same applies to map voting

Reply:
Will create another PR for this